### PR TITLE
Aircraft - Fix ACE_Gatling_30mm_Sub class

### PR DESCRIPTION
**Pull request by PabstMirror**
ref https://github.com/acemod/ACE3/pull/7118#discussion_r365496944

### CODE REVIEW

File: addons/aircraft/CfgAmmo.hpp [hunk 1]
```diff
@@ -75,14 +75,14 @@ class CfgAmmo {
         submunitionAmmo = "ACE_Gatling_30mm_AP_Plane_CAS_01_F";
         ACE_caliber = 1.4;
    };
-   
+
     class ACE_Gatling_30mm_Sub_CM41: ACE_Gatling_30mm_Sub_HEI {
-        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F",0.8,"Cannon_30mm_HE_Plane_CAS_01_F",0.2};
+        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F",0.8,"Gatling_30mm_HE_Plane_CAS_01_F",0.2};
```
Comment (bux) [inline addons/aircraft/CfgAmmo.hpp]:
```suggestion
        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F", 0.8, "Gatling_30mm_HE_Plane_CAS_01_F", 0.2};
```

File: addons/aircraft/CfgAmmo.hpp [hunk 2]
```diff
@@ -75,14 +75,14 @@ class CfgAmmo {
...
         ACE_caliber = 1.4;
     };
 
     class ACE_Gatling_30mm_Sub_CM51: ACE_Gatling_30mm_Sub_HEI {
-        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F",0.83,"Cannon_30mm_HE_Plane_CAS_01_F",0.17};
+        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F",0.83,"Gatling_30mm_HE_Plane_CAS_01_F",0.17};
```
Comment (bux) [inline addons/aircraft/CfgAmmo.hpp]:
```suggestion
        submunitionAmmo[] = {"ACE_Gatling_30mm_AP_Plane_CAS_01_F", 0.83, "Gatling_30mm_HE_Plane_CAS_01_F", 0.17};
```

**Comment by commy2** [comment]
Like this formatting more, bux :)

**Comment by bux** [comment]
but that's not a macro :(

**Pull request merged**